apis/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -9582,6 +9582,8 @@ spec:
                 additionalProperties:
                   type: string
                 type: object
+              notificationsBusInstance:
```

Comment:
@gibizer @bogdando is there a reason behind the fact that the other patch calls this field `notificationsBus` instead of `notificationsBusInstance` like we did in the service operators?

Reply:
ack, then let's keep it consistent with service operators and call it `notificationsBusInstance`.

Reply:
+1 for consistency